spec/immutable-data-structures.html

```diff
@@ -83,6 +83,10 @@ <h1>Record.prototype</h1>
         <p>This property has the attributes { [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: false }.</p>
       </emu-clause>
     </emu-clause>
+    <emu-clause id="sec-properties-of-record-instances">
+      <h1>Properties of Record Instances</h1>
```

Comment:
Is "Record Instance" defined anywhere? Should we call it "Record Object" instead?

Reply:
Yeah that does sound good. I was following how the spec around `Symbol` was written https://tc39.es/ecma262/multipage/fundamental-objects.html#sec-properties-of-symbol-instances where it has this same oddity that `Symbol Object` is defined. But when describing properties they are referred to as instances.

Reply:
It looks like the "instance" wording is used specifically because they inherit from the prototype (so `instanceof` returns true).